extras/tests/MemoryUsageTest/MemoryUsageTest.ino

```diff
@@ -0,0 +1,246 @@
+// MemoryUsageTest.ino
+//
+// Memory Usage unit tests.  Expected to run on Arduino UNO (ATMega328)
+// (tests may fail on other boards)
+//
+// REQUIRES: MemoryUsage, AUnit libraries.
+//
+#include <MemoryUsage.h>
+
+#include <AUnit.h>
```

Comment:
Note this new MemoryUsageTest.ino sketch depends on the AUnit library as well as MemoryUsage.  Only this sketch's tests need AUnit.  Legacy code should still work fine.